frontend_tests/node_tests/realm_playground.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+const {strict: assert} = require("assert");
+
+const {zrequire} = require("../zjsunit/namespace");
+const {run_test} = require("../zjsunit/test");
+
+const pygments_data = zrequire("../generated/pygments_data.json");
+
+const rp = zrequire("realm_playground");
```

Comment:
Let's import this as `const realm_playground = ...`. While we have some exceptions in some test files, I think including the existing `th` one for `typeahead_helper` that you used in a previous commit, I think those abbreviations are a bad practice... Since they mean that `git grep realm_plaground.initialize` isn't a correct way to find all the calls to a function from outside its defining module.

@andersk would it be easy to do a sweep+lint rule against renaming modules when importing them?